public/index.html

```diff
@@ -165,15 +165,21 @@
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
+    <!-- Hotjar Tracking Code for https://www.kaffeerudel.at -->
+    <script>
+      (function (h, o, t, j, a, r) {
+        h.hj =
+          h.hj ||
+          function () {
+            (h.hj.q = h.hj.q || []).push(arguments);
+          };
+        h._hjSettings = { hjid: 1805187, hjsv: 6 };
```

Comment:
Don't agree. Should stay absolutely compact.

Reply:
That would improve readability without affecting performance because the code is compressed during the build process anyway.

So I see no reason why no newline should be added here. Is there one?

Reply:
Ups. No sorry that's incorrect. That is a html file, therefore it is not compressed by default. 
NVM.. Looks good 🚀